### PR TITLE
ci: added jira-sync workflows

### PR DESCRIPTION
**Pull request by farouktelicent**
Added jira-sync workflows to enable syncing issues reported on github to jira

### CODE REVIEW

**Comment by danieldaviestelicent** [comment]
Looks good to me, but I've not done this before so added @rvesse as a reviewer as well to see if he can spot any potential issues.

**Review by danieldaviestelicent — DISMISSED** [review]
gtg as long as Rob is happy too.

File: .github/workflows/jira-sync.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  issues: 
+    types:
+      - opened
+      - edited
+      - transferred
+      - reopened
+
+jobs:
+  sync-to-jira:
+    uses: telicent-oss/shared-workflows/.github/workflows/jira-sync.yml@jira-sync
```
Comment (rvesse) [inline .github/workflows/jira-sync.yml:14]:
```suggestion
    uses: telicent-oss/shared-workflows/.github/workflows/jira-sync.yml@main
```
Should target the `main` branch version of the workflow

**Review by rvesse — COMMENTED** [review]
One minor change needed but otherwise looks good

File: .github/workflows/jira-sync.yml (hunk 1 above)
Comment (farouktelicent) [reply .github/workflows/jira-sync.yml]: Thanks Rob

**Review by rvesse — APPROVED** [review]

**Review by danieldaviestelicent — APPROVED** [review]

**Pull request merged**